### PR TITLE
Checkbox for recommendations/suggestions page.

### PR DESCRIPTION
**Pull request by martinnj**
Used for (de)selecting all the suggested or recommended mods with one
click.

### CODE REVIEW

File: GUI/Main.cs [hunk 1]
```diff
@@ -1042,6 +1042,19 @@ private void FocusMod(string key, bool exactMatch)
...
         }
+
+        private void RecommendedModsToggleCheckbox_CheckedChanged(object sender, EventArgs e)
+        {
+          var state = ((CheckBox)sender).Checked;
+          foreach (ListViewItem item in RecommendedModsListView.Items)
+          {
+            if (item.Checked != state)
+            {
+              item.Checked = state;
```
Comment (Postremus) [inline GUI/Main.cs]: Formating is broken

**Comment by Postremus** [comment]
related to #1148

**Comment by Postremus** [comment]
Tested, works.
Merging this, since formating issues are of minor concern imo.
Thank you for your patience.

**Comment by martinnj** [comment]
Oh shit, sorry for the misalign. Hadn't taught Atom proper indentation. Sorry.

**Comment by martinnj** [comment]
I'll fix it with the old PR, if I can get it to work ;)

**Pull request merged**